normalize.js

```diff
@@ -111,11 +126,8 @@ define(function() {
     
     return out.substr(0, out.length - 1);
   };
-  
-  var normalizeCSS = function(source, fromBase, toBase) {
 
-    fromBase = removeDoubleSlashes(fromBase);
-    toBase = removeDoubleSlashes(toBase);
+  var normalizeCSS = function(source, fromBase, toBase) {
```

Comment:
I see, that `var normalizeCSS` line is not proper aligned (1 space needed). Please update/amend/squash commit.